app/views/layouts/application.html.slim

```diff
@@ -5,6 +5,7 @@ html
     meta charset="utf-8"
     meta content="width=device-width, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0" name="viewport"
     link href='https://fonts.googleapis.com/css?family=Roboto:400,100,700' rel='stylesheet' type='text/css'
+    script src="//code.jquery.com/jquery-2.1.4.min.js" 
```

Comment:
we already have jquery in Gemfile `gem 'jquery-rails'` so this one is unnecessary